library/cwm/src/modules/CWMServiceStart.rb

```diff
@@ -21,16 +21,13 @@
 # you may find current contact information at www.novell.com
 #
 # ***************************************************************************
-# File:	modules/CWMServiceStart.ycp
-# Package:	Common widget manipulation, service start widget
-# Summary:	Routines for service start widget handling
-# Authors:	Jiri Srain <jsrain@suse.cz>
-#
-# $Id$
-#
+
 require "yast"
 
 module Yast
+  # Routines for service start widget handling.
+  #
+  # NOTE: a modern API for this is {::UI::ServiceStatus}
```

Comment:
what about marking it as deprecated and replaced by that?

Reply:
Well, I wasn't confident about that, but since that time I had to look into systemctl anyway, so I'll check again and add more docs about how to migrate to the new API.